src/main/scala/it/pagopa/interop/authorizationmanagement/api/impl/PurposeApiServiceImpl.scala

```diff
@@ -167,16 +167,60 @@ final case class PurposeApiServiceImpl(
         updateAgreementState204
       case Failure(ex) =>
         logger.error(
-          s"Error updating Agreement with EService $eServiceId and Consumer $consumerId state for all clients",
+          s"Error updating Agreement with EService $eserviceId and Consumer $consumerId state for all clients",
           ex
         )
         val problem =
-          problemOf(StatusCodes.InternalServerError, ClientAgreementStateUpdateError(eServiceId, consumerId))
+          problemOf(StatusCodes.InternalServerError, ClientAgreementStateUpdateError(eserviceId, consumerId))
         complete(problem.status, problem)
     }
 
   }
 
+  /**
+   * Code: 204, Message: Clients updated
+   * Code: 400, Message: Client not found, DataType: Problem
+   */
```

Comment:
Mere curiosity, are this generated by IntelliJ?

Reply:
Let me check if this feature can be switched off.